src/views/document/utils/boxes/MapBox.vue

```diff
@@ -21,15 +21,22 @@
 
     <elevation-profile :document="document" v-if="documentType=='outing'" />
 
-    <div
-      v-if="document.geometry && (document.geometry.geom_detail || documentType == 'waypoint')"
-      class="buttons is-centered">
-      <button class="button is-primary is-small" @click="downloadGpx">
-        GPX
-      </button>
-      <button class="button is-primary is-small" @click="downloadKml">
-        KML
-      </button>
+    <div class="columns is-multiline is-mobile is-clearfix">
```

Comment:
Can you provide a screenshot of this? When we change UI, we must be discussed on forum (otherwise, we face big difficulties :) )